llvm/lib/SYCL/ASFixer/LLVMBuild.txt

```diff
@@ -1,21 +0,0 @@
-;===- ./lib/SYCL/LLVMBuild.txt ------------------------------*- Conf -*-----===;
```

Comment:
I added them from a merge with `sycl/unified/master` to avoid loosing https://github.com/triSYCL/sycl/pull/75
But I have no idea whether they are required or not anymore.

Reply:
the file structure was recently flattened to be more inline with the rest of LLVM and to make it easier to add new passes.
so these files aren't useful anymore because there directory is empty.